webapp/src/main/webapp/themes/wilma/templates/page-home.ftl

```diff
@@ -26,89 +26,103 @@
-
-        <!-- List of randomly selected academic departments -->
-        <@lh.academicDeptsHtml />
-
-        <#if geoFocusMapsEnabled >
-            <!-- Map display of researchers' areas of geographic focus. Must be enabled in runtime.properties -->
-            <@lh.geographicFocusHtml />
-        </#if>
-
-        <!-- Statistical information relating to property groups and their classes; displayed horizontally, not vertically-->
-        <@lh.allClassGroups vClassGroups! />
-
-        <#include "footer.ftl">
-        <#-- builds a json object that is used by js to render the academic departments section -->
-        <@lh.listAcademicDepartments />
-    <script>
-        var i18nStrings = {
-            researcherString: '${i18n().researcher?js_string}',
-            researchersString: '${i18n().researchers?js_string}',
-            currentlyNoResearchers: '${i18n().currently_no_researchers?js_string}',
-            countriesAndRegions: '${i18n().countries_and_regions?js_string}',
-            countriesString: '${i18n().countries?js_string}',
-            regionsString: '${i18n().regions?js_string}',
-            statesString: '${i18n().map_states_string?js_string}',
-            stateString: '${i18n().map_state_string?js_string}',
-            statewideLocations: '${i18n().statewide_locations?js_string}',
-            researchersInString: '${i18n().researchers_in?js_string}',
-            inString: '${i18n().in?js_string}',
-            noFacultyFound: '${i18n().no_faculty_found?js_string}',
-            placeholderImage: '${i18n().placeholder_image?js_string}',
-            viewAllFaculty: '${i18n().view_all_faculty?js_string}',
-            viewAllString: '${i18n().view_all?js_string}',
-            viewAllDepartments: '${i18n().view_all_departments?js_string}',
-            noDepartmentsFound: '${i18n().no_departments_found?js_string}'
-        };
-        // set the 'limmit search' text and alignment
-        if  ( $('input.search-homepage').css('text-align') == "right" ) {
-             $('input.search-homepage').attr("placeholder","${i18n().limit_search} \u2192");
-        }
-    </script>
+        <div class="container">
+            <@lh.facultyMemberCount  vClassGroups! />
+            <#include "identity.ftl">
+
+            <#include "menu.ftl">
+
+            <div class="row">
+                <section id="intro" role="region" class="col-md-7">
+                    <h2>${i18n().intro_title}</h2>
+
+                    <p>${i18n().intro_para1}</p>
+                    <p>${i18n().intro_para2}</p>
+
+                    <section id="search-home" role="region">
+                        <h3>${i18n().intro_searchvivo} <span class="search-filter-selected">filteredSearch</span></h3>
+
+                        <fieldset>
+                            <legend>${i18n().search_form}</legend>
+                            <form id="search-homepage" action="${urls.search}" name="search-home" role="search" method="post" >
```

Comment:
It's been a while since I added this code, I can't remember why I switched to the POST method... Also, I see that every other theme is using GET.
I reverted it to GET.